release-notes/opensearch-release-notes-2.6.0.md

```diff
@@ -0,0 +1,539 @@
+# OpenSearch and OpenSearch Dashboards 2.6.0 Release Notes
+
+## Release Highlights
+
```

Comment:
The OpenSearch 2.6.0 release adds tools and enhancements to help you build, manage, and scale OpenSearch for a range of use cases. This release includes a new data schema for analytics and observability workloads as well as new functionality for machine learning-powered search, along with new index management capabilities, enhanced threat detection for security analytics, expanded functionality for map visualizations, and more. See below for the latest additions to the project and visit our downloads page to get started with the new distributions. You can explore OpenSearch Dashboards on the Playground, with no need to download.